src/GafferScene/MapProjection.cpp

```diff
@@ -163,16 +163,19 @@ IECore::ConstObjectPtr MapProjection::computeProcessedObject( const ScenePath &p
 	M44f objectMatrix = inPlug()->fullTransform( path );
 	M44f objectToCamera = objectMatrix * cameraMatrix.inverse();
 
-	CameraPtr camera = constCamera->copy();
-	camera->addStandardParameters();
-	float tanFOV = -1;
-	if( camera->parametersData()->member<StringData>( "projection" )->readable() == "perspective" )
+	bool perspective = constCamera->getProjection() == "perspective";
+
+	Box2f normalizedScreenWindow;
+	if( constCamera->hasResolution() )
```

Comment:
I think we pretty well have to take resolution into account here, because the convention we've discussed is that when you set up a projection camera you bake film fit and resolution onto it.  

Does it create an expectation that could confuse people when using cameras that aren't baked?  Yeah, maybe.  But that feels like a lower priority than making sure we actually get things right for projection cameras.